systest/scripts/install_lbaasv2.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -ex
+
+GLOBAL_ROUTED_MODE=False
```

Comment:
You're right..   this also means I need to change more tlc files...
(at least one)...